typings/config.d.ts

```diff
@@ -52,6 +64,36 @@ declare global {
         instance: InstanceType<typeof Gatherer>;
         options?: {};
       } | string;
+
+      export interface CategoryJson {
+        name: string;
+        description: string;
+        audits: CategoryMemberJson[];
+      }
+
+      export interface GroupJson {
+        title: string;
+        description: string;
+      }
+
+      /**
+       * A LH Artifacts object, but the traces and devtoolsLogs are replaced
```

Comment:
can u add a comment that this is present to support a legacy usecase. and link to the docs

was confused for a sec why runner artifacts were in here. :)

Reply:
done

Reply:
can we agree to nuke this use case before v3? it's a lot of config complexity for no known customers other than our tests

Reply:
yup. we discussed yesterday that this is basically solved via 
`settings: { auditMode: 'path/to/artifacts/' }`

patrick, that in line with your config/settings harmonization?

Reply:
yup exactly 👍